test/distributed/_composable/test_fully_shard.py

```diff
@@ -444,5 +444,53 @@ def test_float16_on_one_submodule(self):
             self.assertEqual(forward_inputs[c2].dtype, torch.float16)
 
 
+class TestFSDPModelCheckpointing(FSDPTest):
+    """Tests composable FSDP model checkpointing."""
+
+    @property
+    def world_size(self) -> int:
+        return 2
+
+    @skip_if_lt_x_gpu(2)
+    def test_state_dict_root_fully_shard(self):
```

Comment:
nit: might want to mention this is full state dict for now.

Reply:
In the line below:
"Tests that the full state dict"